subsys/net/openthread/Kconfig

```diff
@@ -12,10 +12,21 @@ if NET_L2_OPENTHREAD
 
 menu "OpenThread"
 
+config NORDIC_SECURITY_PROMPTLESS
+	bool
+	default y if !CUSTOM_OPENTHREAD_SECURITY
+
+choice OPENTHREAD_SECURITY
+	depends on !OPENTHREAD_SECURITY_INTERNAL
+
+config OPENTHREAD_NRF_SECURITY_CHOICE
```

Comment:
If I am not mistaken also nrf/samples/openthread/common/overlay-ot-defaults.conf file needs to be updated. Currently it states:
# Disable vanilla mbedtls in favor of custom backends
CONFIG_OPENTHREAD_MBEDTLS=n

But I think that now it should select the OPENTHREAD_NRF_SECURITY_CHOICE.
All Thread nrf samples should use nRF security.

Reply:
correct and thanks, extra commit pushed.